src/App.jsx

```diff
@@ -0,0 +1,29 @@
+import React, { useState } from "react";
+import './App.css';
+import { Counter } from './Components/Counter/Counter';
+import { GenreSelect } from './Components/GenreSelect/GenreSelect';
+import { SearchForm } from './Components/SearchForm/SearchForm';
+
+function App() {
+  const [selectedGenre, setSelectedGenre] = useState('All');
+  const genres = ['All', 'Comedy', 'Horror', 'Crime'];
+
+  return (
+    <div className="App">
```

Comment:
`camelCase` is usually used for class names:

 ```suggestion
     <div className="app">
```